src/RoadRunnerBridge/HttpFoundationWorker.php

```diff
@@ -188,7 +179,141 @@ private function timeFloat(): float
     private function stringifyHeaders(array $headers): array
     {
         return array_map(static function ($headerValues) {
-            return array_map(static fn ($val) => (string) $val, (array) $headerValues);
+            return array_map(static fn($val) => (string)$val, (array)$headerValues);
         }, $headers);
     }
+
+    /**
+     * Basically a copy of BinaryFileResponse->sendContent()
+     * @param BinaryFileResponse $response
+     * @return \Closure
+     */
+    private function createFileStreamGenerator(BinaryFileResponse $response): \Closure
+    {
+        return static function () use ($response) {
+            $ref = new \ReflectionClass($response);
+            $maxlen = $ref->getProperty("maxlen")->getValue($response);
```

Comment:
The closure binding is something that I have yet to use, since I always reach for reflections. I will have to test them both and see if there's a major difference.